### ISSUE TITLE
larger batch size hurts accuracy?

### ISSUE THREAD

**Issue by Kewenjing1020** [ifzhang/FairMOT#220]
Hi, I tried to train with a larger batch size (64,96) and a larger learning rate, but the mota and IDF1 get worse. It seems larger batch size always gets better accuracy in other tasks right?

**Comment by ifzhang** [comment]
Sorry, I haven't tried a larger batchsize yet. You may try a smaller learninig rate when using a large batchsize.
